pkgs/misc/emulators/dosbox-staging/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, lib, fetchFromGitHub, autoconf, autoconf-archive, automake, pkgconfig, alsaLib, SDL2, SDL2_net, libGL, libGLU, libogg, opusfile }:
+
+stdenv.mkDerivation rec {
+  pname = "dosbox-staging";
+  version = "v0.75.0";
+
+  src = fetchFromGitHub {
+    owner = "dreamer";
+    repo = pname;
+    rev = version;
+    sha256 = "00n5k2ya7ml55wvrk40p9il01f9k2q2b5g9b20n5zvbgv0d8c5ps";
+  };
+
+  hardeningDisable = [ "format" ];
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ autoconf autoconf-archive automake SDL2 libGL libGLU libogg opusfile alsaLib SDL2_net ];
+
+  preConfigure = "./autogen.sh";
```

Comment:
I'm really bad with `automake` and can never remember (nor care to...) but you might want to investigate `autoreconfHook` as it may (or may not...) be useful in your `nativeBuildInputs` rendering this line obsolete :thinking: